x-pack/plugins/lens/public/datatable_visualization/expression.test.tsx

```diff
@@ -105,8 +140,7 @@ describe('datatable_expression', () => {
             value: 'shoes',
           },
         ],
-        negate: true,
```

Comment:
This is indicating a real bug - the value should be filtered out but instead it's only filtering for this value.
To reproduce:
* Go to Lens
* Create data table
* Filter out value (clicking on the "-" button appearing when hovering over a cell)
* Filter gets created filtering only for that value